docs/streaming-programming-guide.md

```diff
@@ -1162,7 +1162,7 @@ joinedStream = stream1.join(stream2)
 {% endhighlight %}
 </div>
 </div>
-Here, in each batch interval, the RDD generated by `stream1` will be joined with the RDD generated by `stream2`. You can also do `leftOuterJoin`, `rightOuterJoin`, `fullOuterJoin`. Furthermore, it is often very useful to do joins over windows of the streams. That is pretty easy as well. 
```

Comment:
try to minimize changes not directly related to the renaming to make reviewing easier

Reply:
Apologies, my IDE is really determined to remove these trailing spaces (even after I turned the feature off!). I'll be more careful with this.